packages/preview/grotesk-cv/1.0.3/src/template/info.toml

```diff
@@ -0,0 +1,69 @@
+[personal]
+first_name = "Miles"
+last_name = "Dyson"
+profile_image = "portrait.png"
+language = "en"
+include_icons = false
+
+[personal.info]
+address = "Los Angeles, CA"
+telephone = "+1 (555) 123-4567"
+
+[personal.info.email]
+link = "mailto:m.dyson@skynet.ai"
+label = "m.dyson@skynet.ai"
+
+[personal.info.linkedin]
+link = "https://linkedin.com/in/milesdyson"
+label = "linkedin.com/in/miledyson"
+
+[personal.info.github]
+link = "https://github.com/skynetguy"
+label = "github.com/skynetguy"
+
+[personal.icon]
+address = "house"
+telephone = "phone"
+email = "envelope"
+linkedin = "linkedin"
+github = "github"
+homepage = "globe"
+
+[section.icon]
+education = "graduation-cap"
+experience = "briefcase"
+languages = "language"
+other_experience = "wrench"
+personal = "brain"
+profile = "id-card"
+references = "users"
+skills = "cogs"
+
+# To change fill_color to white, replace value with "#ffffff"
+[layout]
+fill_color = "#f4f1eb"
+paper_size = "a4"
+accent_color = "#d4d2cc"
+# deafult value is 71%
+left_pane_width = "71%"
+
+[layout.text]
+font = "Hanken Grotesk"
+size = "10pt"
+cover_letter_size = "11pt"
+
+[layout.text.color]
+light = "#ededef"
+medium = "#78787e"
+dark = "#3c3c42"
+
+[language.en]
+subtitle = "Software Engineer with a knack for human-friendly AI solutions"
+
+[language.es]
+subtitle = "Ingeniero de software con un talento para soluciones de IA amigables para humanos"
+
+[import]
+path = "../../lib.typ"
+lib-path = "../lib.typ"
+fontawesome = "@preview/fontawesome:0.5.0"
```

Comment:
I'm really not convinced by having this in the TOML file, in my opinion it should be directly in the code. In any case, the `lib-path` needs to be fixed to use a `@preview/...` import.